Zotero/Scenes/Detail/ItemDetail/ViewModels/ItemDetailActionHandler.swift

```diff
@@ -777,7 +778,8 @@ struct ItemDetailActionHandler: ViewModelActionHandler, BackgroundDbProcessingAc
                     state.hideController = true
                 }
             }
-            return
```

Comment:
This `return` shouldn't be removed, there's no need to update the viewModel again below, since we're leaving the screen (`state.hideController = true`).

Reply:
In that case, then I can put this piece of code in the last case of `viewModel.state.type`

Reply:
Sure, that works too